cmd/deploy.go

```diff
@@ -409,11 +409,24 @@ func getChartValues(chartValues map[string]interface{}, clusterName string, depl
 		return nil, err
 	}
 
+	agent, ok := chartValues["agent"].(map[string]interface{})
```

Comment:
I think it is better to build a toleration map, and then adding it to the `defaultChartValues` object instead of writing directly to `chartValues`